auth/basic_auth.py

```
@@ -0,0 +1,97 @@
+            self._save_users()
+            return True
+        return False
+
+    def enforce_password_reset(self, username: str) -> None:
+        """Enforce password reset for a user"""
+        if username in self.users:
+            self.users[username]['password_reset_required'] = True
+            self._save_users()
+
+# Initialize auth
+auth = BasicAuth()
```

Flake8 (7.2.0)

[error] expected 2 blank lines after class or function definition, found 1

(E305)